extras/ds18b20/ds18b20.h

```diff
@@ -0,0 +1,20 @@
+#ifndef DRIVER_DS18B20_H_
+#define DRIVER_DS18B20_H_
+
+typedef struct DSENSOR {
+	uint8_t id;
+    uint8_t major;
+    uint8_t minor;
+} DSENSOR;
```

Comment:
The esp-open-rtos convention we generally follow is to name typedefs in the style `name_t`. Could you please change this name to something like `ds18b20_sensor_t`?

You can omit the name immediately after typedef struct, ie something like:

```
typedef struct {
  etc;
} ds18b20_sensor_t;
```

... if you think the name is too long, maybe `ds_sensor_t`?